### PR TITLE
coded front screen

### PR DESCRIPTION
**Pull request by accttakaya**
# What
- codede front screen (see below)
   "Gemfile","Gemfile.lock" for installing 'front-awesome-sass'
   "app/assets/stylesheets/modules/_messages.scss" and the other 7 files for conding front screen

# why
- to code front screen
- to study how haml scss work on rails
- to study BEM(Block, Element, Modifier)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  get 'messages/index'
```
Comment (miyagit) [inline config/routes.rb]: こちらrootがあるので、削除しましょう。

Comment (accttakaya) [reply config/routes.rb]: 修正しました。

**Comment by ken-nag** [comment]
lgtm

**Pull request merged**